static/app.js

```diff
@@ -62,6 +62,8 @@ oppiaFoundationWebsite.run([
   }
 ]);
 
+oppiaFoundationWebsite.constant('ADMIN_EMAIL_ADDRESS', 'admin@example.com');
```

Comment:
Is this used in both frontend + backend? If so it's a good candidate for a constants.js file that both of them share.

Reply:
Yes, it is shared by both frontend and backend. 

I think I will TO-DO the `constants.js` file and the necessary infrastructure for both frontend + backend till later iteration.
1) We are in the finishing sprint toward releasing v1 of the foundation site.
2) This PR should only concern itself with design feedbacks. Setting up services, infrastructures, tests, etc... seem out of scope for this PR imo.

Reply:
OK, sounds good. Would it be possible to file an issue for it? I'm a little uneasy with leaving it like this, because of the DRY principle, but I'm OK with it if we fix it shortly after.

Reply:
Looks like I will just do the `constants.js` PR right after this PR goes in. Since I only have the "Main page" left and I am also waiting for Rachel feedback on the hero image thing before proceeding.